packages/kbn-pm/src/production/build_production_projects.ts

```diff
@@ -125,7 +129,7 @@ async function copyToBuild(project: Project, kibanaRoot: string, buildRoot: stri
   await copy(['**/*', '!node_modules/**'], buildProjectPath, {
     cwd: project.getIntermediateBuildDirectory(),
     dot: true,
-    nodir: true,
+    onlyFiles: true,
```

Comment:
I am not seeing this option [here](https://github.com/sindresorhus/cpy/tree/v8.0.0).

Reply:
Yeah they just pass the options directly to the underlying glob library https://github.com/mrmlnc/fast-glob#onlyfiles

Reply:
Yuck.